packages/zoe/src/contractSupport/ratio.js

```diff
@@ -195,20 +195,30 @@ export const invertRatio = ratio => {
 export const addRatios = (left, right) => {
   assertIsRatio(right);
   assertIsRatio(left);
-  assert(
-    left.numerator.brand === left.denominator.brand &&
-      left.numerator.brand === right.numerator.brand &&
-      left.numerator.brand === right.denominator.brand,
-    X`all brands must match:  ${q(left)} ${q(right)}`,
```

Comment:
I would be surprised to find that code expects an exception even if the operation can produce correct ratios.  That, to me, would indicate a bug in the reliant code.

Reply:
I'd be surprised too. Since we're pre-1.0, I don't feel much obligation to support code outside our own. So this is mostly a somewhat theoretical / forward-looking concern.